network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -89,39 +94,83 @@ <h2 class="h3-heading">{% trans "What is required to sign up?" %}</h2>
           {% trans "Phone"               as phone %}
           {% trans "Third party account" as third_party_account %}
 
-          {% include "fragments/signup_requirement.html" with title=email                 value=product.signup_requires_email          %}
-          {% include "fragments/signup_requirement.html" with title=phone                 value=product.signup_requires_phone          %}
-          {% include "fragments/signup_requirement.html" with title=third_party_account   value=product.signup_requires_third_party_account    %}
+          {% if product.signup_requires_email %}
+            {% include "fragments/signup_requirement.html" with title=email                 value=product.signup_with_email          %}
+          {% endif %}
+          {% if product.signup_requires_phone %}
+            {% include "fragments/signup_requirement.html" with title=phone                 value=product.signup_with_phone          %}
+          {% endif %}
+          {% if product.signup_requires_third_party_account %}
+            {% include "fragments/signup_requirement.html" with title=third_party_account   value=product.signup_with_third_party    %}
+          {% endif %}
```

Comment:
These are not boolean values.
```suggestion
          {% include "fragments/signup_requirement.html" with title=email                 value=product.signup_with_email          %}
          {% include "fragments/signup_requirement.html" with title=phone                 value=product.signup_with_phone          %}
          {% include "fragments/signup_requirement.html" with title=third_party_account   value=product.signup_with_third_party    %}
```